xhttp/xhttpserver/server.go

```diff
@@ -99,18 +98,6 @@ func New(o Options, l *zap.Logger, h http.Handler) Interface {
 			o.Address,
 			l,
 		),
-
-		ConnContext: func(ctx context.Context, c net.Conn) context.Context {
```

Comment:
http.Server.ConnContext is called before the tls connection is established, meaning themis' request builder RequestBuilderFunc(setConnectionState) will not find the context embedded tls ConnectionState.